### PR TITLE
Chore/rad 135 obfuscate header query body in reqest logger

### PR DESCRIPTION
**Pull request by eMKoded**

### CODE REVIEW

File: services/gateway/src/middleware/request-logger.ts [hunk 1]
```diff
@@ -3,6 +3,8 @@ import * as morgan from "morgan";
 import { Request } from "express";
 import { appConfig, MorganFormatTypes } from "../config/config";
 
+const qs = require("qs");
```
Comment (sethii) [inline services/gateway/src/middleware/request-logger.ts:6]: We dont use require, use import

File: services/gateway/package.json [hunk 2]
```diff
@@ -20,9 +20,10 @@
     "helmet": "3.22.0",
     "http-proxy-middleware": "1.0.3",
     "http-status-codes": "1.4.0",
-    "node-fetch": "2.6.0",
     "lodash": "4.17.19",
     "morgan": "1.10.0",
+    "node-fetch": "2.6.0",
+    "qs": "^6.9.4",
```
Comment (mkopa) [inline services/gateway/package.json:26]: remove tilde from semver for consistency

File: services/gateway/src/middleware/request-logger.ts [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import * as _ from "lodash";
 import * as morgan from "morgan";
+import * as qs from "qs";
```
Comment (mkopa) [inline services/gateway/src/middleware/request-logger.ts:3]: Can't we use the built-in `querystring` module?

Comment (eMKoded) [reply services/gateway/src/middleware/request-logger.ts:3]: No, it doesn't work with nested object.

**Pull request merged**